person_frame.rb

```diff
@@ -0,0 +1,25 @@
+require './name_able'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @id = rand(1..1000)
+  end
```

Comment:
- [ ] Kindly make your constructor method something like this shown below :arrow_down:

```ruby
def initialize(age, name = 'Unknown', parent_permission)
    @id = Random.rand(1..999_999_999)
    @name = name
    @age = age
    @parent_permission = parent_permission
    @rentals = []
    super()
  end
```

Reply:
I did it.